assets/constants.ts

```diff
@@ -6602,6 +6602,15 @@ export default {
         }
       ]
     },
+    "COLLECTION_PLAYER": {
+      "ROUTE": "collection/:collection_id",
+      "TITLE": "",
```

Comment:
Any special reason that the title is missing?

Reply:
The title is provided at the time of publishing the collection. If the collection is not published and title is not provided then it is empty only.